modules/launcher/src/main/assembly/assembly.xml

```diff
@@ -49,6 +49,14 @@
             </includes>
             <fileMode>644</fileMode>
         </fileSet>
+        <fileSet>
+            <directory>src/main/resources/</directory>
+            <outputDirectory>resources</outputDirectory>
```

Comment:
Shouldn't the output directory be set to dbscripts?

Reply:
fixed.